classes/class-wc-connect-nux.php

```diff
@@ -83,5 +91,272 @@ public function register_add_service_to_zone_pointer( $pointers ) {
 			);
 			return $pointers;
 		}
+
+		public function get_jetpack_install_status() {
+			// check if Jetpack is activated
+			if ( ! class_exists( 'Jetpack_Data' ) ) {
+				// not activated, check if installed
+				if ( 0 === validate_plugin( 'jetpack/jetpack.php' ) ) {
+					return self::JETPACK_INSTALLED;
+				}
+				return self::JETPACK_UNINSTALLED;
+			} else if ( defined( 'JETPACK_DEV_DEBUG' ) && true === JETPACK_DEV_DEBUG ) {
+				// installed, activated, and dev mode on
+				return self::JETPACK_DEV;
+			}
+
+			// installed, activated, dev mode off
+			// check if connected
+			$user_token = Jetpack_Data::get_access_token( JETPACK_MASTER_USER );
+			if ( isset( $user_token->external_user_id ) ) { // always an int
+				return self::JETPACK_CONNECTED;
+			}
+
+			return self::JETPACK_ACTIVATED;
+		}
+
+		private function should_display_nux_notice_on_screen( $screen ) {
+			if ( // Display if on any of these admin pages.
+				( // Products list.
+					'product' === $screen->post_type
+					&& 'edit' === $screen->base
+				)
+				|| ( // Edit product page if at least one product already published
```

Comment:
I spoke with @olaolusoga about this since the meeting, and we decided to add it to the products page when there's more than 1 published product already, since it does not detract from the intended WC NUX experience at that point.

At the meeting we discussed specifically that showing the banner on the 'Edit Product' page would be detrimental on the first product, but not subsequent products.